Bender.yml

```diff
@@ -26,6 +26,8 @@ dependencies:
   register_interface:       { git: "https://github.com/pulp-platform/register_interface",     rev: 50270f7    }  # TODO: master commit; use next release once out
   riscv-dbg:                { git: "https://github.com/pulp-platform/riscv-dbg",              version: 0.8.0  }
   serial_link:              { git: "https://github.com/pulp-platform/serial_link",            version: 1.1.0  }
+  clic:                     { git: "https://github.com/pulp-platform/clic",                   rev: bed98f8    }  # TODO: branch xnxti-2; merge
```

Comment:
At lease, these should be master commits before merge. Ideally, both would be releases.